openproblems/data/nestorowa2016.py

```diff
@@ -0,0 +1,40 @@
+import os
+import tempfile
+
+import scprep
+import scanpy as sc
+
+from .utils import loader, filter_genes_cells
+
+URL = "https://ndownloader.figshare.com/files/25555751"
+
+
+@loader
+def load_nestorowa_2016(test=False):
```

Comment:
Ideally the dataset name would give a little more information on what the data is.

Reply:
fixed!